packages/@coorpacademy-progression-engine/src/types.js

```diff
@@ -8,9 +8,22 @@ export type Step = {
 
 export type Answer = Array<string>;
 
+export type CHAPTER = 'chapter';
+export type LEVEL = 'level';
+export type SLIDE = 'slide';
+export type NODE = 'node';
+export type FAILURE = 'failure';
+export type SUCCESS = 'success';
+export type ContentType = CHAPTER | LEVEL | SLIDE | NODE | FAILURE | SUCCESS;
+
+export type ContentSlide = {|
+  type: SLIDE,
+  ref: string
+|};
+
 export type GenericContent = {
```

Comment:
If you can, and if it's mandatory for the mobile starts needs, please use strict for these types. Else, don't take care of this comment :)

Reply:
✅  --> some types are to intricated (State, Slide)
for many it's ok